skimage/color/colorlabel.py

```diff
@@ -73,7 +73,8 @@ def _match_label_with_color(label, colors, bg_label, bg_color):
 
 @change_default_value("bg_label", new_value=0, changed_version="0.19")
 def label2rgb(label, image=None, colors=None, alpha=0.3,
-              bg_label=-1, bg_color=(0, 0, 0), image_alpha=1, kind='overlay'):
+              bg_label=-1, bg_color=(0, 0, 0), image_alpha=1, kind='overlay',
+              saturation=0):
```

Comment:
Shouldn't `saturation` be a keyword only argument?
```suggestion
              *, saturation=0):
```

Reply:
Why?

Reply:
Sorry, this question was for @scikit-image/core :). We usually make new arguments keyword only...

Reply:
Yes, I think `saturation` should be a keyword-only argument. And should we raise a warning if it is used while `kind='avg'?

I would also expect a code unit later on to raise an error or, at least, a warning if `saturation < 0` or `saturation > 1`.

Reply:
Just added a warning to check the saturation correct range (also a unit test) and `saturation` to be a keyword-only argument

Reply:
Other parameters like `image_alpha`, `alpha`, etc are not used when `kind='avg'` and there is no warning